tests/bears/BearTest.py

```diff
@@ -396,16 +398,18 @@ def test_download_cached_file(self):
         filename = self.filename
         file_location = self.file_location
 
-        with requests_mock.Mocker() as reqmock:
+        with freeze_time('2017-01-01 00:00:00.00') as frozen_datetime, \
+                requests_mock.Mocker() as reqmock:
+
             reqmock.get(mock_url, text=mock_text)
             self.assertFalse(isfile(file_location))
             expected_filename = file_location
             result_filename = self.uut.download_cached_file(mock_url, filename)
             self.assertTrue(isfile(join(file_location)))
             self.assertEqual(result_filename, expected_filename)
             expected_time = getmtime(file_location)
-            sleep(0.5)
 
+            frozen_datetime.tick(delta=datetime.timedelta(seconds=0.5))
```

Comment:
We have mocked datetime to `2017-01-01 00:00:00.00` and this will change frozen time +500ms. So we do not need `sleep` anymore.